Adding the tetra-metrics functionality, we compute the quality and ed…

Gengis-cyber opened the pull request:
…ge lengths of a tetrahedron and we display it in the terminal and in the rendering window. Just need to press shift + clic on a tetrahedron.

Gengis-cyber committed:
Adding the tetra-metrics functionality, we compute the quality and edge lengths of a tetrahedron and we display it in the terminal and in the rendering window.

Gengis-cyber committed:
Fixing the quality : adding the alpha coefficient (12√3) to mormalize the quality to 1 for a regular tetrahedron

mpotse commented:
Thanks to Gengis, who made this change at my request. I suggest not to consider a merge right away, this branch may still need a little cleanup.

Gengis-cyber committed:
Cleaning some comments and fixing a double comment that I did not see before
for the triangles picking.
Also a better display of the messages in the terminal.

Gengis-cyber committed:
Deleting the messages "Displaying..." and "No tetrahedron object selected" because they are saying the obvious.

Gengis-cyber committed:
Deleting the compil.date, because it is generated by cmake.

Gengis-cyber committed:
Better organization of the code, putting some functions in a more suitable place.

Gengis-cyber committed:
Deleting a block that is not really necessary anymore.
